cla_frontend/apps/zendesk/views.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"Zendesk"
+import json
+from django.http import HttpResponse
+from django.views.generic.base import View
+
+from cla_frontend.apps.zendesk.client import zendesk_client
+
+
+class ZendeskView(View):
+
+    def post(self, request):
+        "Create a new Zendesk ticket"
+        response = zendesk_client.post_feedback(
```

Comment:
More of a style thing than a real issue. `post_feedback` says what you are doing, but not why. `create_ticket` would be a more intent-revealing method name.